services/kanto/include/kanto_service.h

```diff
@@ -0,0 +1,25 @@
+#ifndef KANTO_SERVICE_H_
```

Comment:
Some of your files have a copyright statement, some are missing it.

Reply:
added licenses to all source files now